### DISCUSSION TITLE
`vitest run` vs `vitest --run` (and same for `watch`)

### DISCUSSION THREAD

**Discussion by segrey** [vitest-dev/vitest#2394] [General]
Could you please clarify which cli is more preferable: `vitest run` or `vitest --run`?
Same question for `vitest watch` vs `vitest --watch`.
It's unclear from `vitest --help`.
Since these options are duplicated, are there any plans to remove the duplicates?

Any insights would be appreciated. Thanks!

**Comment by sheremet-va** [comment]
They are the same

**Reply by segrey** [reply]
Thanks! Am I right that both will be supported in future?

**Reply by sheremet-va** [reply]
We don't have plans to remove them. But maybe we should better document it.
